typescript/src/schema/proto.ts

```diff
@@ -436,6 +442,35 @@ function augmentService(
       '.google.api.oauthScopes'
     ].split(',');
   }
+  augmentedService.pathTemplate = [];
+  for (const property of Object.keys(messages)) {
+    const m = messages[property];
+    if (m && m.options) {
+      const option = m.options;
+      if (option && option['.google.api.resource']) {
+        const opt = option['.google.api.resource'];
+        const onePathTemplate = option[
```

Comment:
OK, let's do `opt as ResourceDescriptor` then :)  It raises questions when you access `options['.google.api.resource'] twice in the two consecutive lines.